spire-controller-manager.yaml

```diff
@@ -0,0 +1,57 @@
+package:
+  name: spire-controller-manager
+  version: 0.6.0
+  epoch: 0
+  description: A Kubernetes Controller manager which facilitates the registration of workloads and establishment of federation relationships.
+  copyright:
+    - license: Apache-2.0
+
+environment:
```

Comment:
you don't need these, so it can be removed.

Reply:
@developer-guy the entire environment block should be removed ?